lib/ddtrace/contrib/sinatra/tracer.rb

```diff
@@ -71,52 +76,30 @@ def render(engine, data, *)
             end
           end
 
+          app.use HeaderTaggerMiddleware
```

Comment:
You renamed the file but didn't rename the constant?

Reply:
I trusted RubyMine too much it seems. Sorry for that.

Reply:
Interestingly the tests have passed 😱

Reply:
Sorry for that, some brain meltdown happened